src/pages/landing/stocks/_dtrading.js

```diff
@@ -0,0 +1,169 @@
+        width: 100%;
+        max-width: 58.8rem;
+        margin: 0 auto;
+    }
+`
+
+const ImageWrapper = styled.div`
+    display: flex;
+    width: 40%;
+    margin-right: ${(props) => props.margin_right};
+
+    @media ${device.tabletL} {
+        margin: 2rem auto;
+        max-width: 58.8rem;
+        width: 100%;
+    }
+`
+const StyledHeader = styled(Header)`
+    line-height: 1.25;
+
+    @media ${device.tabletL} {
+        font-size: 40px;
+        line-height: 40px;
+        margin-top: 2rem;
+        text-align: center;
+    }
+`
+const Row = styled.div`
+    flex-direction: ${(props) => props.flex_direction};
+    width: 100%;
+    display: flex;
+    margin-top: 4rem;
+
+    &:first-child {
+        margin-top: 0;
+    }
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+    }
+`
+const query = graphql`
+    query {
+        dbot_strategy: file(relativePath: { eq: "dbot-strategy.png" }) {
+            ...fadeIn
+        }
+        dbot_build_strategy: file(relativePath: { eq: "dbot-build-strategy.png" }) {
+            ...fadeIn
+        }
+        dbot_maximise_profits: file(relativePath: { eq: "dbot-maximise-profits.png" }) {
+            ...fadeIn
+        }
+        dbot_track_your_performance: file(relativePath: { eq: "dbot-track-your-performance.png" }) {
+            ...fadeIn
+        }
+        dbot_get_integrated_help: file(relativePath: { eq: "dbot-get-integrated-help.png" }) {
+            ...fadeIn
+        }
+        dbot_save_your_strategies: file(relativePath: { eq: "dbot-save-your-strategies.png" }) {
+            ...fadeIn
+        }
+        dmt_5_ipad_iphone: file(relativePath: { eq: "dmt-5-ipad-iphone.png" }) {
+            ...fadeIn
+        }
+        dmt_5_mac: file(relativePath: { eq: "dmt-5-mac.png" }) {
+            ...fadeIn
+        }
+        make_smarter: file(relativePath: { eq: "make-smarter.png" }) {
+            ...fadeIn
+        }
+        trade_what_you_want: file(relativePath: { eq: "trade-what-you-want.png" }) {
+            ...fadeIn
+        }
+        how_you_want: file(relativePath: { eq: "how-you-want.png" }) {
+            ...fadeIn
+        }
+        stocks_zero_fees: file(relativePath: { eq: "stocks-zero-fees.png" }) {
+            ...fadeIn
+        }
+        stocks_blue_chip: file(relativePath: { eq: "stocks-blue-chip.png" }) {
+            ...fadeIn
+        }
+    }
+`
+const DTrading = ({ trading, reverse, two_title }) => {
+    const data = useStaticQuery(query)
+    return (
+        <StyledSection>
+            <StyledContainer>
+                {trading.map((item, index) => {
+                    let is_even = reverse ? (index + 1) % 2 : index % 2
+                    return (
+                        <Row flex_direction={!is_even ? 'row' : 'row-reverse'} key={index}>
+                            <Content
+                                margin_right={!is_even ? '4.0rem' : '0'}
+                                margin_left={!is_even ? '0' : '4.0rem'}
```

Comment:
```suggestion
                                mr={!is_even ? '4.0rem' : '0'}
                                ml={!is_even ? '0' : '4.0rem'}
```

if using `Flex` component

Reply:
Changed.